components/Nav/NavMobile/NavMobile.js

```diff
@@ -59,6 +58,13 @@ export default class NavMobile extends Component {
                   </Link>
                 </li>
               ))}
+              {isLoggedIn && (
+                <li className={styles.li}>
+                  <button type="button" className={styles.button} onClick={logout}>
```

Comment:
Now I understand this.

I think we can get around this manipulation by making Logout a link that goes to `/login?loggedOut=true`

Then, on the login page, check for that query param in context and do the necessary unauth.

With that done:
- The abstraction pattern within the nav area is maintained.
- Every item in the nav is truly a link
- We can easily dynamically render a success Alert letting the user know they've logged out

Reply:
Love it